CMakeLists.txt

```diff
@@ -1,6 +1,16 @@
-﻿cmake_minimum_required (VERSION 3.16)
+﻿include(cmake/version.cmake)
```

Comment:
Mettre l'include après le cmake_minimum

Reply:
done in [bd8a153](https://github.com/deltacasttv/video-viewer/pull/4/commits/bd8a153a3880e54e7f9a1f4a8e1a1e83d97d5284)